game.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-#define BOARD_SIZE 4
+#define BOARD_SIZE 3
```

Comment:
It seems like you changed the default setting again. I think we would like to keep the default setting to a 4x4 board size.